crates/voicevox_core/src/error.rs

```diff
@@ -5,11 +5,7 @@ use super::*;
 use std::path::PathBuf;
 use thiserror::Error;
 
-/*
- * 新しいエラーを定義したら、必ずresult_code.rsにあるVoicevoxResultCodeに対応するコードを定義し、
- * internal.rsにある変換関数に変換処理を加えること
- */
-
+/// エラー。
```

Comment:
何も書かないよりは、と思ったのですがこれくらいにはした方がよかったかも。

```suggestion
/// VOICEVOX COREのエラー。
```

Reply:
たしかに。`VOICEVOX COREのエラー。`、良いと思いました！

Reply:
ではこれをcommit suggestionします。